posts/2021-09-07-java17-210010-beta.adoc

```diff
@@ -0,0 +1,238 @@
+---
+layout: post
+title: "Java 17 beta support, JWE support for access and ID tokens, MicroProfile Context Propagation 1.3, and other enhancements in Open Liberty 21.0.0.10-beta"
```

Comment:
Update all titles to:
Java 17 early access support, JWE support for access and ID tokens, MicroProfile Context Propagation 1.3, and more in Open Liberty 21.0.0.10-beta